backend/Tournament/models.py

```diff
@@ -0,0 +1,52 @@
+from django.db import models
+from django.utils import timezone
+
+from User.models import User
+from .tournamentState import TournamentState
+
+# Create your models here.
+class Tournament(models.Model):
+	host_user = models.ForeignKey(User, related_name='tournament_hosts', on_delete=models.CASCADE)
+	name = models.CharField(max_length=30, null=True, blank=True, default=None)
+	state = models.IntegerField(choices=TournamentState.choices, default=TournamentState.LOBBY)
+	next_match = models.IntegerField(default=0)
+	player_amount = models.PositiveIntegerField(null=False, blank=False)
+	winner = models.ForeignKey(User, related_name='tournament_winners', on_delete=models.CASCADE, null=True, blank=True, default=None)
+
+	insert_ts = models.DateTimeField(auto_now_add=True)
+	start_ts = models.DateTimeField(null=True, blank=True)
+	end_ts = models.DateTimeField(null=True, blank=True)
+	updated_ts = models.DateTimeField(auto_now=True)
+
+	expires_ts = models.DateTimeField(null=True, blank=True)
```

Comment:
Isnt this token stuff, a tournament doesnt really expire - or am I missing something?

Reply:
We also kind of break again the naming rule, why have insert_ts and then update_ts and suddenyl expire**S**_ts